modules/installation-restricted-network-samples.adoc

```diff
@@ -64,7 +64,7 @@ $ oc image mirror registry.redhat.io/rhscl/ruby-25-rhel7:latest ${MIRROR_ADDR}/r
 configuration object:
 +
 ----
-$ oc create configmap registry-config --from-file={MIRROR_ADDR}=$path/ca.crt -n openshift-config
+$ oc create configmap registry-config --from-file={MIRROR_ADDR}..5000=$path/ca.crt -n openshift-config
```

Comment:
In step 2 we have used `${MIRROR_ADDR}`, it's better to use `${MIRROR_ADDR_HOSTNAME}..5000` to distinguish them.
And, it's better to give examples about those variables